docs/ansible_collection.md

```diff
@@ -15,7 +15,7 @@ Kubespray can be installed as an [Ansible collection](https://docs.ansible.com/a
    collections:
    - name: https://github.com/kubernetes-sigs/kubespray
      type: git
-     version: v2.23.1
+     version: master # use the appropriate tag or branch for the version you need
```

Comment:
HI @VannTen 

The docs files will auto-sync to the docs site: https://kubespray.io/#/docs/ansible_collection?id=usage
As a docs reader, he may follow the docs step by step. 
Using a released version is better than a master branch, so the `v2.23.1` may be fine for them.

Best Regards :-)

Reply:
HI @VannTen 

What do you think of the comments?
It is better to merge the PR before the release. :-)

Reply:
HI @VannTen 

I create a PR for the comments, https://github.com/VannTen/kubespray/pull/1 . 

Would you please help to review it :-)

Reply:
I would rather avoid specifying a particular tag, because the user still needs to think of **what version** they need rather than going blindly through the step.
Besides, we already use placeholders in that doc (`INVENTORY` and `PLAYBOOK` in step 4).
IMO, we should either leave `master` or replace it with `BRANCH/GIT_REF` (or something of the sort)

Reply:
@VannTen  Thanks a lot, I agree with it.

/approve